<launch>

  <node name="soundplay_node" pkg="sound_play" type="soundplay_node.py" output="screen"/>
  
  <node name="recognizer" pkg="pocketsphinx" type="recognizer.py" output="screen" clear_params="true">
    <param name="lm" value="$(find jr_speech)/config/3dv_demo/3dv_commands.lm"/>
    <param name="dict" value="$(find jr_speech)/config/3dv_demo/3dv_commands.dic"/>
    <!-- Use the command 'pacmd list-sources | grep alsa_input' to get microphone names -->
    <!--param name="mic_name" value="alsa_input.usb-Plantronics_Plantronics_BT300M_263993CCC7D14F6787021589130D6328-00-BT300M.analog-mono" /-->
  </node>

  <node pkg="jr_speech" type="speech_demo.py" name="speech_demo" clear_params="true" output="screen">
     <remap from="target_topic" to="/face_recognizer/face_recognitions" />
     
     <param name="allowed_phrases" value="$(find jr_speech)/config/3dv_demo/3dv_commands.txt"/>
     <param name="nav_config_file" value="$(find jr_navigation)/config/3dv/locations.yaml" />
     
     <rosparam>
        rate: 10
        max_target_distance: 2.0
        lost_detection_timeout: 5.0
        tts_voice: voice_don_diphone
     </rosparam>
     
  </node>
   
</launch>
